### PR TITLE
Text Reader Improvement Added

### LINKED ISSUE #1172: [Feature Request]:  Text Reader Improvement
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Text Reader Improvement

### Use Case

Text Reader Improvement

### Benefits

Text Reader Improvement

### Add ScreenShots

Text Reader Improvement
https://apu52.github.io/METAVERSE/Projects/Text-Reader/

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [ ] I'm a SSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Damini2004**
# Title and Issue number 

Title : Text Reader Improvement Added

Issue No. : #1172 

Code Stack : HTML, CSS

Close #1172

# Video (mandatory)

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have gone through the  `contributing.md` file before contributing

**Additional context (Mandatory )**

***Are you contributing under any Open-source programme?***

- [x] I'm a GSSOC'24 contributor
- [ ] I'm a SSOC'24 contributor

### CODE REVIEW

**Comment by Damini2004** [comment]
#1210 #1211 Kindly assign this to me. @apu52

File: Projects/Text-Reader/index.html [hunk 1]
```diff
@@ -3,10 +3,11 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Text Reader</title>
+    <title><strong><u>Text Reader</u></strong></title>
```
Comment (apu52) [inline Projects/Text-Reader/index.html:6]: why this <strong><u> present here in the name , it looks odd

**Review by apu52 — CHANGES_REQUESTED** [review]
please do the necessary changes , otherwise the other changes are really good

File: Projects/Text-Reader/index.html (hunk 1 above)
Comment (apu52) [reply Projects/Text-Reader/index.html:6]: image

**Comment by Damini2004** [comment]
@apu52 Changes resolved.

**Review by apu52 — APPROVED** [review]
@Damini2004 good work

**Pull request merged**